lib/fixed_length/parser.rb

```diff
@@ -0,0 +1,42 @@
+module FixedLength
+  class Parser
+    DEFAULT_TYPE = 'a'
+
+    attr_reader :columns, :body
+
+    def initialize(columns, body)
+      @columns = columns
+      @body = body
+    end
+
+    def parse
+      zip unpacked_body
+    end
+
+
+    private
+
+    def unpacked_body
+      body.lines.map do |line|
+        line.unpack(unpack_format)
```

Comment:
Yea, we don't really love this implementation - we saw it for the first time yesterday actually which is why I pinged @jakehow. It works, yes, but I'm not sure it's the best tool for this job.

Since each row in the bank data file is a string, where its semantic components are column delimited (i.e. the routing number is `bank_row_string[0, 9]` or the first nine digits or the row/string), it might be easier to just index in to the string on the fly.